src/django_keycloak/backends.py

```diff
@@ -17,7 +16,7 @@ class KeycloakAuthenticationBackend(RemoteUserBackend):
     def authenticate(
         self,
         request,
-        username: Optional[str] = None,
+        remote_user: Optional[str] = None,
```

Comment:
This makes it a bit confusing, as `remote_user` can now be username or Keycloak's internal user ID. We should at least document what the method expects in `remote_user`.

Reply:
This is due to linting warnings, due to you renaming parent method arguments name. If they pass arguments as keyword you will not catch it.

Reply:
But I agree with documenting

Reply:
Added in [c32c78c](https://github.com/urbanplatform/django-keycloak-auth/pull/39/commits/c32c78cd20e0488e04b6d7c5420742d0df3f644c)